sqs/user_template.go

```diff
@@ -179,3 +182,45 @@ func NewRolePolicyDocument(resources, actions []string) PolicyDocument {
 		},
 	}
 }
+
+func GetCannedAccessPolicy(policyName string) ([]string, error) {
+	switch policyName {
+		case "full":
```

Comment:
style: could we create a new type for these policies as well as predefined the keys, so we can lean on the type system a bit...

ie.

```
type AccessPolicy = string

var (
    AccessPolicyFull AccessPolicy = "full"
    AccessPolicyProducer AccessPolicy = "producer"
    AcessPolicyConsumer AccessPolicy = "consumer"
)
```

This will make it a little easier to jump around the code to see where some string came from and make it harder to pass an invalid string